Updated with steps to overcome an issue on macOS (#274)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc

--
You might need to grant permissions to run the file depending on your operating system and settings.

[NOTE]
====

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 

Reviewer: ```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to `Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 
```

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 

Reviewer: ```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the MacOS' approach towards security and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 
```

Author: @spolti , per Apple[1] [2], it is macOS since 2016..

[1] - https://www.apple.com/sg/macos/ventura/
[2] - https://en.wikipedia.org/wiki/MacOS

Reviewer: @bbalakriz didn't understand your comment :)
I am not saying it isn't on mac since.... I just suggested a very small rephrase :)

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 

Reviewer: what and where Bin is located?

Author: @spolti, that message is thrown by macOS and a macOS user typically knows where a Bin is located...

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeage https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. 

Reviewer 2: I am assuming the word is overage.
```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overage https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this.  
```

Author: Sorry, it was supposed to be "overeager".

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this. 

Reviewer: ```suggestion
If you are using MacOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this. 
```

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this. 

Reviewer: ```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager https://support.apple.com/en-us/HT202491[approach towards security] by MacOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this. 
```

> If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager https://support.apple.com/en-us/HT202491[approach towards security] by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this. 

Reviewer: ```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is because MacOS has a particularly strict approach towards  https://support.apple.com/en-us/HT202491[security]. To resolve this issue, you can move the "KIE Sandbox Extended Services" application out of quarantine. You can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command to achieve this. 
```

> If you are using macOS, you might encounter the error `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overzealous approach towards security by macOS and could be fixed by moving the "KIE Sandbox Extended Services" application out of quarantine. An example command to do that is `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app`. Please refer this https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/[article] for more details around this. 

Reviewer 2: ```suggestion
If you are using macOS, you might encounter the `"KIE Sandbox Extended Services" is damaged and can't be opened. You should move it to Bin` while running the application "KIE Sandbox Extended Services". This is due to the overeager approach towards security by macOS and you can fix it by moving the "KIE Sandbox Extended Services" application out of quarantine. To achieve this, you can execute the `xattr -d com.apple.quarantine /Applications/KIE\ Sandbox\ Extended\ Services.app` command. For more information, see https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/[article].
```

Reviewer 3: I personally would vote to don't link solutions from articles that are not official from apple or other trusted sources on our official docs. Maybe we can write similar content from the website in our docs.

The real solution for this would be to sign our binaries.
